search_gov_crawler/search_gov_spiders/settings.py

```diff
@@ -111,3 +115,30 @@
     "http": "scrapy_playwright.handler.ScrapyPlaywrightDownloadHandler",
     "https": "scrapy_playwright.handler.ScrapyPlaywrightDownloadHandler",
 }
+
+now = datetime.now()
+date_time = now.today().isoformat()
+
+SPIDERMON_ENABLED = True
+SPIDERMON_MIN_ITEMS = 1000
+SPIDERMON_TIME_INTERVAL = 1  # time is in seconds
+SPIDERMON_ITEM_COUNT_INCREASE = 100
+SPIDERMON_MAX_EXECUTION_TIME = 86400 
+SPIDERMON_UNWANTED_HTTP_CODES_MAX_COUNT = 10
+SPIDERMON_UNWANTED_HTTP_CODES = [400, 407, 429, 500, 502, 503, 504, 523, 540, 541]
+SPIDERMON_REPORT_TEMPLATE = "results.jinja"
+SPIDERMON_REPORT_CONTEXT = {"report_title": "Spidermon File Report"}
+SPIDERMON_REPORT_FILENAME = f"{date_time}_spidermon_file_report.html"
+SPIDERMON_EMAIL_SUBJECT = "Spidermon report"
```

Comment:
I think this message could be customized in a future store, for instance to have the environment name in it and/or the spider name.  What do you think?

Reply:
The spider name makes sense! For the environment name- I thought we were only going to have this run in production?